jerry-debugger/jerry_client_main.py

```diff
@@ -1151,33 +1163,34 @@ def _process_incoming_text(self, buffer_type, data):
                 log_type = "%sout:%s " % (self.blue, self.nocolor)
 
                 message = self.current_out + message
-                lines = message.split("\n")
-                self.current_out = lines.pop()
+                lines = message.decode("utf8").split("\n")
+                self.current_out = lines.pop().encode("utf8")
 
                 return "".join(["%s%s\n" % (log_type, line) for line in lines])
 
             if subtype == JERRY_DEBUGGER_OUTPUT_DEBUG:
                 log_type = "%slog:%s " % (self.yellow, self.nocolor)
 
                 message = self.current_log + message
-                lines = message.split("\n")
-                self.current_log = lines.pop()
+                lines = message.decode("utf8").split("\n")
+                self.current_log = lines.pop().encode("utf8")
 
                 return "".join(["%s%s\n" % (log_type, line) for line in lines])
 
             if not message.endswith("\n"):
```

Comment:
This check will fail if the message is a byte string. So we'll need to decode the string just before this.

Reply:
Thanks, I've applied you change from https://gist.github.com/galpeter/a6ac851b6ab113aaad62d60001c5df23.